pyproject.toml

```diff
@@ -84,7 +84,7 @@ trio = "*"
 fakeredis = "^1.9.4"
 
 [tool.poetry.extras]
-testing = ["httpx", "cryptography"]
+testing = ["httpx"]
```

Comment:
i think we can eliminate the testing extra - and have httpx installed as part of the core package, hence have the test client as a builtin and also simplify the export / import paths (we can then export from the root `__init__.py` file)

Reply:
Could you elaborate on simplifying the export / import paths? You mean just re-export `TestClient` and `create_test_client` from `starlite/__init__.py`?

Reply:
Yup. 

This is possible if httpx is not an extra

Reply:
Done